build_docker_image.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -e
 
-: ${DOCKER_IMAGE:=huginn/huginn}
+: ${DOCKER_IMAGE:=aroberts/huginn}
```

Comment:
Leftover from development? Guessing unintended.
```suggestion
: ${DOCKER_IMAGE:=huginn/huginn}
```
Otherwise, this looks great!